make.js

```diff
@@ -560,6 +564,16 @@ target.bump = function() {
         }
 
         taskJson.version.Patch = taskJson.version.Patch + 1;
+        taskLocJson.version.Patch = taskLocJson.version.Patch + 1;
+
         fs.writeFileSync(taskJsonPath, JSON.stringify(taskJson, null, 4));
+        fs.writeFileSync(taskJsonPath, JSON.stringify(taskLocJson, null, 2));
```

Comment:
This should be taskLocJsonPath not taskJsonPath

Reply:
Fixed.